src/librustc/session/config.rs

```diff
@@ -1117,6 +1117,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "make unnamed regions display as '# (where # is some non-ident unique id)"),
     emit_end_regions: bool = (false, parse_bool, [UNTRACKED],
         "emit EndRegion as part of MIR; enable transforms that solely process EndRegion"),
+    include_user_assert_ty: bool = (false, parse_bool, [UNTRACKED],
+        "include UserAssertTy as part of MIR dumps"),
```

Comment:
I think we should just *always* include it...?

Reply:
Will remove this flag in a future commit.

Reply:
Removed this.